### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**Build Product Listing page**

- Navigation Component
- Filter products Component
- Products container Component
- Axios Get Products data from Backend db 
- Render data in Products container

### CODE REVIEW

File: src/backend/db/products.js [hunk 1]
```diff
@@ -8,23 +8,107 @@ import { v4 as uuid } from "uuid";
...
+    fastDelivery: true,
+    ratings: 3.5,
+  },
+  {
+    _id: uuid(),
+    categories: "Women",
+    image: "/Products-images/long-cote-Women.webp",
+    brand: "Roadster",
+    name: "Dark Blue long Jacket",
+    discoutPrice: 5999,
```
Comment (SukanyaSen530) [inline src/backend/db/products.js:107]: Discounted price can be calculated based on actual price and discount.

File: src/components/productListing/ProductsFilters.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@700&family=Quicksand:wght@300&family=Roboto:wght@100;300;400;500&display=swap");
```
Comment (SukanyaSen530) [inline src/components/productListing/ProductsFilters.css:1]: If this font is being used through out the application, can be moved to index.css

File: src/backend/db/products.js (hunk 1 above)
Comment (tanishk15sharma) [reply src/backend/db/products.js:107]: yes , this is just dummy price

File: src/components/productListing/ProductsFilters.css (hunk 2 above)
Comment (tanishk15sharma) [reply src/components/productListing/ProductsFilters.css:1]: yes , you are right , but this font style is specifically used here only.

**Pull request merged**